SeventhHeavenUI/ViewModels/MainWindowViewModel.cs

```diff
@@ -1129,7 +1129,9 @@ internal static bool SaveActiveProfile()
                 using (FileStream fs = new FileStream(Sys.PathToCurrentProfileFile, FileMode.Create))
                 {
                     Util.Serialize(Sys.ActiveProfile, fs);
-                }
+                }
+                IEnumerable<string> profileDetails = Sys.ActiveProfile.GetDetails();
+                File.WriteAllLines(Path.Combine(Sys.PathToProfiles, "profile.txt"), profileDetails);
```

Comment:
Please remove these lines, not needed.

Reply:
Totally, I just thought it looked nicer 😅

Reply:
All good, thanks :)